### PR TITLE
infra: backported run tests.yml workflow on ubuntu runners also for rhel branches

### PR DESCRIPTION
**Pull request by adamkankovsky**
backported: https://github.com/rhinstaller/anaconda/pull/5618

### CODE REVIEW

**Comment by KKoukiou** [comment]
@adamkankovsky the tests failed here, this needs investigation.

**Comment by jkonecny12** [comment]
Current test failure should be handled by different PR. It is failing because our workflow files differs between master and this branch.

**Comment by jkonecny12** [comment]
Failed test will be fixed by: https://github.com/rhinstaller/anaconda/pull/5627

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ concurrency:
 
 jobs:
   unit-tests:
-    runs-on: [self-hosted, kstest]
+    runs-on: ubuntu-20.04
```
[the remark this answers is not in the document]
Comment (KKoukiou) [reply .github/workflows/tests.yml]: ok

**Review by jkonecny12 — APPROVED** [review]
LGTM!

**Comment by KKoukiou** [comment]
Closing in favor of https://github.com/rhinstaller/anaconda/pull/5627

**Pull request closed without merging**